--- origsrc/cogl-1.8.2/cogl-pango/Makefile.am	2011-08-12 10:32:56.000000000 -0500
+++ src/cogl-1.8.2/cogl-pango/Makefile.am	2011-11-17 02:52:08.838381700 -0600
@@ -32,7 +32,7 @@ libcogl_pango_la_LIBADD = $(top_builddir
 libcogl_pango_la_LIBADD += $(COGL_DEP_LIBS) $(COGL_PANGO_DEP_LIBS) $(COGL_EXTRA_LDFLAGS)
 libcogl_pango_la_LDFLAGS = \
 	-export-dynamic \
-	-export-symbols-regex "^cogl_pango_.*"
+	-export-symbols-regex "^cogl_pango_.*" \
 	-no-undefined \
 	-version-info @COGL_LT_CURRENT@:@COGL_LT_REVISION@:@COGL_LT_AGE@
 
